Source/ULibs.TinyJsonDeser/JsonDeserializer.cs

```diff
@@ -646,15 +646,34 @@ private static bool TryParseObject(string json, ref int offset, out IDictionary<
                     SkipWhiteSpace(json, ref index);
                     if (TryParseValue(json, ref index, out var value))
                     {
-                        output = new KeyValuePair<string, object>(key, value);
+                        output = new KeyValuePair<string, object?>(key!, value);
                         offset = index;
                         return true;
                     }
                 }
             }
 
-            output = default(KeyValuePair<string, object>);
+            output = default;
             return false;
         }
+
+        /// <summary>Specifies that when a method returns <see cref="ReturnValue"/>, the parameter will not be null even if the corresponding type allows it.</summary>
+        [AttributeUsage(AttributeTargets.Parameter, Inherited = false)]
+#if INTERNAL_NULLABLE_ATTRIBUTES
+    internal
+#else
+        public
+#endif

```

Comment:
Should this just always be `internal`?

Reply:
yup now you mention it I remember reading that in one of the articles you linked